src/main/java/org/fundacionjala/sfdc/pageobjects/PageUtil.java

```diff
@@ -0,0 +1,43 @@
+package org.fundacionjala.sfdc.pageobjects;
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.pageobjects.components.TopMenu;
+import org.fundacionjala.sfdc.pageobjects.config.SalesForceAppEnvsConfig;
+import org.openqa.selenium.WebDriver;
+
+public class PageUtil {
+    private static PageUtil instance;
+    private WebDriver webDriver = DriverManager.getNavigator();
+
+    protected PageUtil() {
+        initialize();
+    }
+
+    public static PageUtil getInstance() {
+        if (instance == null) {
+            instance = new PageUtil();
+        }
+        return instance;
+    }
+
+    private void initialize() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)